__tests__/test-data.test.js

```diff
@@ -50,3 +48,35 @@ describe("api/topics", () => {
     );
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("returns article api ", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("200,check if it returns an array ", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const arrayOfArticles = response.body.data;
+        expect(Array.isArray(arrayOfArticles)).toBe(true);
+      });
+  });
+  test("200,returns the correct obj ", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then((response) => {
+        const arrayOfArticles = response.body.data;
+        const articleId = arrayOfArticles[0].article_id;
+        expect(articleId).toBe(2);
+      });
+  });
+
+  test("400, article must be a number ", () => {
+    return request(app).get("/api/articles/j").expect(400);
+  });
+  test("204, no content fouund  ", () => {
```

Comment:
This should be a 404 not a 204, 204 implies no content which would be correct for a delete. A 404 means that there is something missing that has been requested.